src/components/Nav/Nav.js

```diff
@@ -1,7 +1,7 @@
 import styled from 'styled-components/macro'
 import { Link } from 'react-router-dom'
 import { IconContext } from 'react-icons'
-import { MdSportsTennis, MdLeaderboard } from 'react-icons/md'
+import { MdSportsTennis, MdLeaderboard, MdHistory } from 'react-icons/md'
```

Comment:
That's a suggestion Mareike made in my repo that I really liked which I then deployed:
Maybe change the naming of the icons to something like `IconHistory`. Makes the code easier to read

Reply:
Will think about it.

`import { MdHistory as HistoryIcon}`

Like this?

Reply:
Yes!